main.tf

```diff
@@ -76,11 +76,46 @@ module "nodes_api_uat_singapore" {
   }
 }
 
+module "nodes_api_uat_singapore_channels" {
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v2.3.1"
+  env               = "api_uat"
+  envid             = "api_uat"
+  bootstrap_version = var.bootstrap_version
+  vault_role        = "ae-node"
+  vault_addr        = var.vault_addr
+  subnets           = module.nodes_api_uat_singapore.subnets
```

Comment:
I guess this should be in outputs

Reply:
https://github.com/aeternity/terraform-aws-aenode-deploy/blob/master/outputs.tf#L9